deploy_and_run_tests.sh

```diff
@@ -213,11 +213,11 @@ export VIRTUAL_ENV_DISABLE_PROMPT="something"
 virtualenv "${temp_dir}"
 source ${temp_dir}/bin/activate;
 trap clean_up EXIT
+pip install --upgrade .[int_test]
```

Comment:
Thanks for the reminder. The integration tests failed since the unit tests and the integration tests have different version requirements on `google-cloud-bigquery`. Please check on [Pull 244](https://github.com/googlegenomics/gcp-variant-transforms/pull/244) for updates. And thanks a lot for helping out resolving the dependencies offline!